### PR TITLE
Disable app lock if passcode is removed

### PR DESCRIPTION
**Pull request by mikeger**
- Fixed layout on iPad
- Dismiss keyboard if needed

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Overlay/AppLockViewController.swift [hunk 1]
```diff
@@ -109,9 +128,9 @@ import HockeySDK.BITHockeyManager
     }
 
     /// @param callback confirmation; if auth is not needed called with 'true'
-    func requireLocalAuthenticationIfNeeded(with callback: @escaping (Bool)->()) {
+    func requireLocalAuthenticationIfNeeded(with callback: @escaping (Bool?)->()) {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Overlay/AppLockViewController.swift]: What's the reasoning behind making this optional?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Overlay/AppLockViewController.swift]: If the auth is not needed or device does not support it, it supposed to return .none (also I need to update the comment)

File: Wire-iOS/Sources/UserInterface/Overlay/AppLockViewController.swift [hunk 2]
```diff
@@ -170,5 +182,6 @@ extension AppLockViewController: UIApplicationDelegate {
     
     func applicationDidBecomeActive(_ application: UIApplication) {
         self.showUnlockIfNeeded()
+        self.resignKeyboardIfNeeded()
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Overlay/AppLockViewController.swift]: Shouldn't we dismiss the keyboard when going to the background as well? Isn't the keyboard briefly visible when only dismissing it in `applicationDidBecomeActive`?

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Overlay/AppLockViewController.swift]: I updated to dismiss the keyboard before going offscreen too

**Pull request merged**